### PR TITLE
[FIX] Update text to reflect that BackgroundSuppressionPulseTime is recommended

### PR DESCRIPTION
**Pull request by michaelf500**
regarding Issue: Background suppression timing for ASL #1842

I've changed 'required' to 'recommended' and slightly reworded the sentence.

Michael

### CODE REVIEW

File: src/appendices/arterial-spin-labeling.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ For (P)CASL, specifying the `LabelingDuration` and the `PostLabelingDelay` is re
 The `LabelingDuration` is defined as the total duration of the labeling pulse train in seconds.
 `PostLabelingDelay` is the time in seconds after the end of the labeling until the middle of the excitation pulse applied
 to the imaging slab (for 3D acquisition) or first slice (for 2D acquisition).
-Additionally, the `BackgroundSuppressionPulseTime`'s is required in case `BackgroundSuppression` was applied.
+Additionally, the `BackgroundSuppressionPulseTime`'s is recommended in the case where `BackgroundSuppression` was applied.
```
Comment (effigies) [inline src/appendices/arterial-spin-labeling.md:82]:
```suggestion
Additionally, the `BackgroundSuppressionPulseTime` is RECOMMENDED if `BackgroundSuppression` was applied.
```

**Review by effigies — APPROVED** [review]
Agreed. RECOMMENDED/REQUIRED/OPTIONAL should always be all-caps if they are intended to be used in the [RFC 2119](https://www.rfc-editor.org/rfc/rfc2119) sense. Small grammar suggestions that I think read better. Feel free to pick and choose parts of this edit.

**Review by tsalo — APPROVED** [review]
LGTM!

**Comment by tsalo** [comment]
We have two approvals and it's been 5 days (not that the 5-day embargo is required for small bug fixes like this). I'm going to merge now.

**Pull request merged**